package.json

```diff
@@ -59,7 +59,7 @@
     "babel-plugin-version-inline": "^1.0.0",
     "coveralls": "^2.13.0",
     "eslint-plugin-import": "^2.16.0",
-    "eslint-plugin-luma-gl-custom-rules": "file:./eslint",
+    "eslint-plugin-luma-gl-custom-rules": "file:./dev-modules/eslint-plugin-luma-gl-custom-rules",
```

Comment:
Nit: I assume we not planning to publish this module? If we did it would either be part of probe or ocular? In that case I am not sure dev-modules is the right place.

Although I guess we could name it something like `@probe.gl/eslint` so that we publish it on npm under the right name now, then we can move the source later?